sirepo/uri_router.py

```diff
@@ -116,6 +118,11 @@ def init(app, simulation_db):
 
     for n in _REQUIRED_MODULES + tuple(sorted(feature_config.cfg().api_modules)):
         register_api_module(importlib.import_module('sirepo.' + n))
+    for f in pkio.sorted_glob(simulation_db.STATIC_FOLDER.join('en','*.html')):
+        @sirepo.api_perm.allow_visitor
+        def x(filename=f):
+            return sirepo.http_reply.gen_redirect(f'/en/{filename.basename}')
```

Comment:
filename should not be a parameter. It should be the lexically scoped value. Otherwise, I don't think this code will do the right thing, because Flask will pass filename. `f'/en/{filename.basename}'` should be evaluated in the enclosing scope.